dpctl/tensor/_elementwise_common.py

```diff
@@ -213,7 +214,7 @@ def __call__(self, x, /, *, out=None, order="K"):
 
             if res_dt != out.dtype:
                 raise ValueError(
-                    f"Output array of type {res_dt} is needed,"
+                    f"Output array of type {res_dt} is needed, "
```

Comment:
On this line, since space is added after comma, it should be removed before "got" on the next line.